repls.org

```diff
@@ -1078,7 +1078,7 @@
 
   Clojure and Java have provided a solid foundation for developing
   ClojureScript, much of the Clojure infrastructure could be reused, and the
-  Google Closure Compiler, a Java project, has been indispensible for providing
+  Google Closure Compiler, a Java project, has been indispensable for providing
```

Comment:
`Essential` sounds good (to me personally). It is relatively widely used, so it should be fairly understandable.